README.md

```diff
@@ -816,14 +816,19 @@ end
 ```
 
 If you would like to add alternative reporting for the Coverage.report! ouput, you can supply your own by calling add_output in the `ChefSepc::Coverage` block:
+Note the reportOutput has teh following items in it
```

Comment:
@tas50 this was corrected in commit https://github.com/sethvargo/chefspec/pull/822/commits/1b73dcbe500dda12633a4e38c8cd3e282c8edd20